src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -57,6 +58,9 @@ function SearchRouter({onRouterClose}: SearchRouterProps) {
     const [autocompleteSuggestions, setAutocompleteSuggestions] = useState<AutocompleteItemData[] | undefined>([]);
     const [autocompleteSubstitutions, setAutocompleteSubstitutions] = useState<SubstitutionMap>({});
 
+    const isWeb = getPlatform() === CONST.PLATFORM.WEB;
+    const isDesktop = getPlatform() === CONST.PLATFORM.DESKTOP;
```

Comment:
This goes against our best practices - https://github.com/Expensify/App/blob/0dd44ee996d7d2b41ec117205fb52b10a33cf5f0/README.md#platform-specific-file-extensions